open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlUI.java

```diff
@@ -0,0 +1,246 @@
+package io.opensphere.core.control.keybinding;
+
+import io.opensphere.core.util.AwesomeIconSolid;
+import javafx.geometry.Insets;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+
+/**
+ * Creates a table/legend of the current Opensphere shortcut keys which pertain
+ * to General and Map controls.
+ */
+public class ControlUI extends GridPane
+{
+    /**
+     * The directory of the "plus" icon.
+     */
+    private String myPlusDir = "images/keys/plus.png";
```

Comment:
Since this is only used to create an ImageView, cut all the creation time further in the class by making the class variable the ImageView. Could also be final.

Reply:
the problem I ran into was that if used more than once the FX Nodes won't add it because it appears as a duplicate.